static/js/lib/validation/profile_test.js

```diff
@@ -385,15 +385,29 @@ describe("Profile validation functions", () => {
       assert.deepEqual(personalValidation(profile), {})
     })
 
-    it("should complain if you enter an invalid phone number", () => {
-      const profile = {
-        ...USER_PROFILE_RESPONSE,
-        phone_number: "+1 222"
-      }
-      const errors = {
-        phone_number: "Please enter a valid phone number"
-      }
-      assert.deepEqual(personalValidation(profile), errors)
+    //
+    ;[
+      ["+84-03-6123-4567", true],
+      ["+1 222", false],
+      ["asdfasdf", false]
```

Comment:
It would be great if you could add a couple more test values, some US phone numbers as well as the Vietnamese one you've already included. Doesn't add much as far as functionality is concerned, just being verbose with the testing.

Reply:
sure, I'll add a few more - I didn't add too many other than a case that exposed the issue because the external library *should* be tested...